index.js

```diff
@@ -210,6 +210,8 @@ function isSourceElement(opts, node) {
     case "TypeAliasDeclaration": // Typescript
     case "ExportAssignment": // Typescript
     case "ExportDeclaration": // Typescript
+    case "OperationDefinition": // GraphQL
+    case "FragmentDefinition": // GraphQL
```

Comment:
This document is for queries, we also support the schema definitions. I would just look at the printer and take all the ones that end with it